Tell `lsh-pam-checkpw', the PAM password helper program, to use a more
descriptive service name.

--- lsh-2.0.4/src/lsh-pam-checkpw.c	2003-02-16 22:30:10.000000000 +0100
+++ lsh-2.0.4/src/lsh-pam-checkpw.c	2008-11-28 16:16:58.000000000 +0100
@@ -38,7 +38,7 @@
 #include <security/pam_appl.h>
 
 #define PWD_MAXLEN 1024
-#define SERVICE_NAME "other"
+#define SERVICE_NAME "lshd"
 #define TIMEOUT 600 
 
 static int
